### DISCUSSION TITLE
Q: Netlify部署出现Not initialized错误 || Q: Not initialized error occurred during Netlify deployment

### DISCUSSION THREAD

**Discussion by ningguoxu** [walinejs/waline#2793] [Q&A] [answered]
### Describe the bug | 问题描述

Q: Netlify部署出现Not initialized错误

![image](https://github.com/user-attachments/assets/13ccdea7-a2df-4857-a6f8-d8c420af3b3c)

### Website URL | 问题网站

https://glittering-selkie-8505ad.netlify.app/.netlify/functions/comment

### Where your waline deploy? | 服务部署在哪里？

Netlify

### Where your comment data store? | 数据存储在哪里？

TiDB

---
### Describe the bug | Problem description

Q: Not initialized error occurs during Netlify deployment

![image](https://github.com/user-attachments/assets/13ccdea7-a2df-4857-a6f8-d8c420af3b3c)

### Website URL | Problem website

https://glittering-selkie-8505ad.netlify.app/.netlify/functions/comment

### Where your waline deploy? | Where is the service deployed?

Netlify

### Where your comment data store? | Where is the data stored?

TiDB

**Comment by lizheming** [comment, marked as answer]
环境变量没设置对，走到兜底的 LeanCloud 服务去了。这个自行检查下吧。

---
The environment variables were not set correctly, so I went to the hidden LeanCloud service. Check this yourself.
